### PR TITLE
Reland Create /tasks/email-reviewers in notifier

### PR DESCRIPTION
**Pull request by KyleJu**
#2340. Create a /tasks/email-reviewers in notifier. This task receives gate_type as a param and retrieves approvers as the email recipients.

### CODE REVIEW

File: internals/notifier.py [hunk 1]
```diff
@@ -146,16 +146,12 @@ def apply_subscription_rules(
   return results
 
 
-def make_email_tasks(fe: FeatureEntry, is_update: bool=False,
-    changes: Optional[list]=None):
+def make_email_tasks(fe: FeatureEntry, target_emails: list[str],
```
Comment (jrobbins) [inline internals/notifier.py:149]: I think you need to keep this function as-is for the existing email change notifications, and then create a new, separate function to make email tasks for the reviewers.  The reason is that for the reviews, we don't want to notify blink component owners, feature starrers, or anyone else other than the reviewers.

**Review by jrobbins — CHANGES_REQUESTED** [review]
Your approach looks right, but because we don't want to notify feature starrers, etc., we should have more separation between the existing change notification logic and this new reviewer notification logic.

File: internals/notifier.py (hunk 1 above)
Comment (KyleJu) [reply internals/notifier.py]: Done

**Review by jrobbins — APPROVED** [review]
This looks better, thanks for the change.

**Pull request merged**